planetary-social/nos
# Remove wss:// from relay lists and allow entry of relay without wss://

## Issues covered
[1401](https://github.com/planetary-social/nos/issues/1401)

## Description
Made changes to remove the wss:// from relay lists and changes to allow the entry of relays without the wss://. Also modified the relayAddressPlaceholder localizable string.

## How to test
- navigate to the relay list
- observe that there are no longer any wss:// references
- add a relay without using a prepended wss:// and observe that it is added successfully

## Screenshots/Video
- before changes: 
![1401-before](https://github.com/user-attachments/assets/bd289539-6aa0-43a4-b709-5d61ae7f70af)
- after changes: 
![1401-after](https://github.com/user-attachments/assets/3d8fb6a9-3f02-4607-aa6a-2df0be464107)
